src-backend/api/views.py

```diff
@@ -37,6 +39,25 @@ def get_queryset(self):
         user = self.request.user
         return models.Page.objects.filter(procedure__owner_id__exact=user.id)
 
+    @list_route(methods=['PATCH'])
+    def partial_bulk_update(self, request, *args, **kwargs):
+        queryset = self.filter_queryset(self.get_queryset)
+
+        if not request.body:
+            return Response(status=status.HTTP_406_NOT_ACCEPTABLE)
+
+        serializer = self.get_serializer(
+            instance=queryset,
+            data=json.loads(request.body),
+            many=True,
+            partial=True
+        )
+
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
```

Comment:
It is. http://stackoverflow.com/questions/11899088/is-django-post-save-signal-asynchronous